modules/gltf/register_types.cpp

```diff
@@ -142,6 +142,11 @@ void initialize_gltf_module(ModuleInitializationLevel p_level) {
 		GLOBAL_DEF_RST("filesystem/import/fbx/enabled", true);
 		GDREGISTER_CLASS(EditorSceneFormatImporterBlend);
 		GDREGISTER_CLASS(EditorSceneFormatImporterFBX);
+		// Can't (a priori) run external app on these platforms.
+		GLOBAL_DEF_RST("filesystem/import/blender/enabled.android", false);
+		GLOBAL_DEF_RST("filesystem/import/blender/enabled.web", false);
+		GLOBAL_DEF_RST("filesystem/import/fbx/enabled.android", false);
+		GLOBAL_DEF_RST("filesystem/import/fbx/enabled.web", false);
```

Comment:
Ah btw this won't work due to this: #64100.

But when we fix that issue it should work fine.